examples/llama/main.zig

```diff
@@ -159,90 +231,101 @@ pub fn asyncMain() !void {
         .sharding_enabled = true,
     };
 
-    var args = std.process.args();
-    const cli_args = flags.parse(&args, CliArgs);
-    const model_file = cli_args.model;
-
-    var arena_state = std.heap.ArenaAllocator.init(allocator);
-    defer arena_state.deinit();
-    const model_arena = arena_state.allocator();
-
-    const create_opts = try std.json.parseFromSliceLeaky(zml.Platform.CreateOptions, model_arena, cli_args.create_options, .{});
-    const platform = context.autoPlatform(create_opts).withCompilationOptions(compilation_options);
+    // initialize ZML platform with optional create options
+    // eg: --create-options='{"cuda":{"allocator":{"bfc":{"memory_fraction": 0.99}}}}'
```

Comment:
it this working ? I'm not seeing the needed changes in platform.zig

Reply:
it ist. there are create opts in platform.zig and autoPlatform takes them